README.md

```diff
@@ -151,3 +152,22 @@ For example, Juju Dashboard can use the existing controller API to check OpenFGA
 relations. Using that API the dashboard can check the user's relations and
 determine if it should render the `ReBACAdmin` component and associated
 navigation.
+
+## React Query
+
+ReBAC Admin will use the QueryClient from the host application if it exists.
+This allows you to have central control over caching, error handling etc.
+
+**IMPORTANT:** If you've set up React Query you **MUST** set `staleTime` to some
```

Comment:
**Copy-paste of same comment as in the previous PR.**

Just a thought, but wouldn't it be more reliable for us to add a fallback `staleTime` with something like
```
queryClient.setDefaultOptions({
    ...queryClient.getDefaultOptions(),
    queries: {
      ...queryClient.getDefaultOptions().queries,
      staleTime: contextClient?.getDefaultOptions().queries.staleTime ?? 30000,
    },
  });
  ```
instead of relying on the developers to carefully read this part of the docs and set it themselves?

Reply:
This now throws an error. See: https://github.com/canonical/rebac-admin/pull/141#discussion_r1694671384